tests/ldap.js

```diff
@@ -369,5 +369,33 @@ describe('ldap', function() {
       });
     });
   });
+  
+  describe('#usersList', function() {
+    before(function (done) {
+      return done();
+    });
+    after(function (done) {
+      ldap.deleteUser(VALID_USER.username, done);
+    });
+    it('should return a list with one more user than before', function (done) {
+      ldap.getAllUsers(function(err, prevUsers) {
+        expect(err).to.not.exist;
+        var prevCount = prevUsers ? prevUsers.length : 0 ;
```

Comment:
This should be the total numbr of users in the system. The tests need to be self-contained. !fix